src/components/editor/FormWrapper.jsx

```diff
@@ -42,4 +43,9 @@ class FormWrapper extends Component{
   }
 }
 
+FormWrapper.propTypes= {
+  propertyTemplates: PropTypes.array,
```

Comment:
`propertyTemplates: PropTypes.arrayOf(PropTypes.object).isRequired` might be better here ...